crates/sui-core/src/authority.rs

```diff
@@ -981,6 +1008,42 @@ impl AuthorityState {
         }
     }
 
+    #[instrument(level = "trace", skip_all)]
+    pub async fn handle_transaction_v2(
```

Comment:
This doesn't seem to be the right naming anymore since we are only doing various checks here.

Reply:
Do you have an alternate suggestion for name?

I think actually the existing `handle_transaction` was poorly named and should have been called `sign_transaction`. But for the new one I'm adding, `handle` is more apt because we really are handling the full transaction e2e: sending an unsigned tx through consensus, executing it, and returning the effects. I didn't think temporary confusion of naming overload would be too bad, bc eventually we plan to remove the old version.